scripts/ansible/deploy_vanilla.yml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+- hosts: localhost
+  any_errors_fatal: true
+  tasks:
+     - include_tasks: tasks/ansible-install-prereqs.yml
+     - include_tasks: tasks/ansible-install-openenclave-deps.yml
+     - include_tasks: tasks/ansible-install-sgx-driver.yml
+     - include_tasks: tasks/ansible-install-sgx-packages.yml
+     - include_tasks: tasks/ansible-install-openenclave.yml
```

Comment:
I think that's okay. I want to make sure the prereqs and openenclave-deps get installed before the sgx-driver and sgx-packages get installed, just to be safe in case they need some of these build tool dependencies. Since these tasks should be idempotent, it shouldn't be harmful!